src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
yeah, that's correct, and this is something I noticed with a cursor as well... will fix it